app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: ENV["FROM_EMAIL"]
```

Comment:
Also, in order to make the tests pass, we need to set this value `environments/test.rb`.